protocol/errorsHelp.json

```diff
@@ -1,3 +1,14 @@
 {
-  "40103": "https://support.ably.io/solution/articles/3000065731-error-code-40103-invalid-use-of-basic-auth-over-non-tls-transport"
+  "40101": "https://support.ably.io/solution/articles/3000067415-40101-request-mac-does-not-match",
+  "40103": "https://support.ably.io/solution/articles/3000065731-error-code-40103-invalid-use-of-basic-auth-over-non-tls-transport",
+  "40104": "https://support.ably.io/solution/articles/3000068941-40104-timestamp-not-current",
+  "40105": "https://support.ably.io/solution/articles/3000068938-40105-nonce-value-replayed",
+  "40142": "https://support.ably.io/solution/articles/3000081686-error-code-40142-token-expired-code",
+  "40143": "https://support.ably.io/solution/articles/3000081687-error-code-40143-token-unrecognised",
+  "40144": "https://support.ably.io/solution/articles/3000080687-error-code-40144-unexpected-error-decoding-jwt-decode-exception",
+  "40160": "https://support.ably.io/solution/articles/3000081689--error-code-40160-action-not-permitted",
+  "40300": "https://support.ably.io/solution/articles/3000081685--error-code-40300-account-disabled",
```

Comment:
Can we duplicate this one for the additional error codes at github.com/ably/ably-common/pull/37 please?

Reply:
Done @mattheworiordan .